extensions/xkcd/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "xkcd",
+  "title": "xkcd",
+  "description": "Browse the xkcd comics and keep track of the ones you've read.",
+  "icon": "icon.png",
+  "author": "maggie",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "xkcd",
+      "subtitle": "https://xkcd.com",
```

It's up to you, but I think mentioning the service name without link looks cleaner here.
```suggestion
      "subtitle": "XKCD",
```